common/src/main/java/com/ultreon/mods/exitconfirmation/config/Config.java

```diff
@@ -31,6 +32,7 @@ public Config() {
         this.forceTransparentBackground = this.add("screen.forceTransparentBackground", true, "Make the background of the close prompt always transparent.");
         this.allowNarrator = this.add("screen.allowNarrator", false, "Enable the narrator in the close prompt.");
         this.confirmDelay = this.add("screen.confirmDelay", 40, 5, 100, "Confirmation delay in ticks.");
+        this.disconnectPrompt = this.add("prompt.disconnect", true, "Show disconnect prompt when quitting from world or server.");
```

Comment:
Not sure if I want to have it enabled by default...

Reply:
Oh, sure thing. You don't need to accept the pull request at all if you don't think it suits the project haha, I just wanted a mod that prevents me from accidentally leaving servers and there wasn't one. I thought this one had the feature but it didn't.

Reply:
Well, it might get annoying if someone wants to disconnect quickly.
So, if someone needs it they can just enable it.

Reply:
I mean it suits the project, but I don't want to make it an annoyance for some people.
Exitting the game entirely is less of annoyance than exiting a world/disconnecting a server.

Reply:
Depends on the situation yeah, in survival there is no real point but I mostly play on minigame servers and there accidentally disconnecting means losing the game. Leaving it off by default might be the better choice tho

Reply:
> Leaving it off by default might be the better choice tho

Yeah.
It can also be an annoyance if someone wants to trigger something that happens when joining, but you need a precise timing. Then you would need to rejoin in quick succession. But there might be more stuff that might need a quick rejoin.

Reply:
Should I make the default false after merging, or are you going to push a new commit?

Reply:
Changed it :)